cmd/dfdaemon/app/init.go

```diff
@@ -91,7 +91,7 @@ func initLogger(cfg config.Properties) error {
 	return errors.Wrap(dflog.Init(logrus.StandardLogger(), opts...), "init log")
 }
 
-// cleanLocalRepo checks the files at local periodically, and delete the file when
+// cleanLocalRepo checks the files at local periodically, and deletes the file when
```

Comment:
ditto

Reply:
// cleanLocalRepo checks the files at local periodically, and deletes the file when
// it comes to a certain age(counted by the last access time).
// TODO: what happens if the disk usage comes to high level?

Does it really need to be added . after when?

Reply:
I am sorry I missed there is one more line above